frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilderModal/MagicFormBuilderModalContent/PromptDetailsScreen.tsx

```diff
@@ -0,0 +1,170 @@
+  FormErrorMessage,
+  FormLabel,
+  ModalBody,
+  ModalHeader,
+  Text,
+} from '@chakra-ui/react'
+
+import { BxsErrorCircle } from '~assets/icons'
+import Button from '~components/Button'
+import Textarea from '~components/Textarea'
+
+import { useMagicFormBuilderWizard } from '../MagicFormBuilderWizardContext'
+
+export const MagicFormBuilderPromptDetailsScreen = (): JSX.Element => {
+  const {
+    handleDetailsSubmit,
+    formMethods,
+    isLoading,
+    isFetching,
+    handleBack,
+  } = useMagicFormBuilderWizard()
+
+  const {
+    register,
+    setValue,
+    formState: { errors },
+  } = formMethods
+
+  const handleUseIdea = (idea: string) => () => setValue('prompt', idea)
+  const [inputValue, setInputValue] = useState('')
+
+  return (
+    <>
+      <ModalHeader color="secondary.700">
+        <Container maxW={'42.5rem'} p={0}>
+          Write a prompt
+        </Container>
+      </ModalHeader>
+      <ModalBody whiteSpace="pre-wrap">
+        <Container maxW={'42.5rem'} p={0}>
+          <FormLabel>I want to create a form that collects...</FormLabel>
+          <FormControl isRequired isInvalid={!!errors.prompt}>
+            <Textarea
+              {...register('prompt', {
+                required: 'Please enter a prompt.',
+                onChange: (e) => setInputValue(e.target.value),
+                maxLength: {
+                  value: 300,
+                  message: 'Please enter at most 300 characters.',
+                },
+              })}
+              placeholder="personal particulars for an event"
+            />
+            <FormErrorMessage>
+              <FormErrorIcon h="1.5rem" as={BxsErrorCircle} />
+              {errors.prompt?.message} {inputValue.length}/300
+            </FormErrorMessage>
+          </FormControl>
+          <Box mt="1rem">
+            <Flex
+              flexDir="column"
+              border="1px"
+              borderColor="neutral.400"
+              bg="white"
+              mt="-.25rem"
+              py="0.75rem"
+              borderRadius="0.25rem"
+            >
+              <Flex my="0.5rem" px="1rem">
+                {<BiBulb fontSize="1.6rem" />}
+                <Text textStyle="subhead-1" px="1rem">
+                  Try one of these
+                </Text>
+              </Flex>
+              <Button
+                variant="clear"
+                color="secondary.500"
+                justifyContent="flex-start"
+                padding="0"
+                onClick={handleUseIdea(
+                  'Employee feedback on workplace environment and culture.',
+                )}
+              >
+                <Text
+                  textStyle="subhead-1"
+                  color="secondary.500"
+                  fontWeight="400"
+                  px="1rem"
+                  textAlign="justify"
+                >
+                  employee feedback on workplace environment and culture.
+                </Text>
+              </Button>
```

Comment:
the code for each of these prompt ideas are repeated. maybe we can factor them out to an array map instead? e.g. above we can define 

```
  const promptIdeas = [ 'employee feedback on workplace environment and culture.', 'event registrations and dietary preferences for dance ...]
```

 and then below when defining the component we can simply do

```
promptIdeas.map((idea: string) => <Button ....)
```

Reply:
this can also be factored out into the constants file in my other comment :)